client/js/api.js

```diff
@@ -0,0 +1,58 @@
+let baseDomain = 'http://localhost:3000/',
```

Comment:
Please consider environments setup. Base should be configurable. 
Please also consider deploying application to heroku or some other service after.